software/invoiceshelf.yml

```
@@ -0,0 +1,11 @@
+name: "InvoiceShelf"
+website_url: "https://invoiceshelf.com/"
+source_code_url: "https://github.com/InvoiceShelf/InvoiceShelf"
+description: "InvoiceShelf is an open-source software that helps you track expenses, payments & create professional invoices & estimates (fork of Crater)."
```

Shorten the description slightly to get rid of redundant information. 
```suggestion
description: "A software that helps you track expenses, payments & create professional invoices & estimates (fork of Crater)."
```

---

Done @Rabenherz112  - Thanks for the suggestion.